### ISSUE TITLE
Hide empty sections?

### ISSUE THREAD

**Issue by equinusocio** [vuese/vuese#18]
Hi, there is a way to hide empty sections inside the generated readmes? By default it generate empty sections where no comments are found,, like that:

```md
## Events

<!-- @vuese:Icon:events:start -->
<!-- @vuese:Icon:events:end -->

## Slots

<!-- @vuese:Icon:slots:start -->
<!-- @vuese:Icon:slots:end -->

## Methods

<!-- @vuese:Icon:methods:start -->
<!-- @vuese:Icon:methods:end -->
```
<img width="778" alt="screenshot 2019-01-08 at 14 52 27" src="https://user-images.githubusercontent.com/10454741/50834797-41499500-1355-11e9-89c7-74294a69e3a8.png">

Resulting in ugly documentation when a component doesn't have all the data.

**Comment by HcySunYang** [comment]
I just released the patch version: v1.4.2

**Comment by equinusocio** [comment]
So fast. Thank you.
